README.md

```diff
@@ -12,16 +12,20 @@ libraries.
 
 For now, you can find a rendered snapshot at
 [http://aturon.github.io/](http://aturon.github.io/).  After
-[some infrastructure work](https://github.com/aturon/rust-guidelines/issues/17), the snapshot will move somewhere more
-official.
+[some infrastructure work](https://github.com/aturon/rust-guidelines/issues/17), the snapshot will move somewhere more official.
```

Comment:
Ah these are now located here: http://doc.rust-lang.org/style/. We haven't published this link too too much though as they're only somewhat more official just yet, we haven't quite gone through the guidelines and agreed as a community that they're what we all agree on (and this may take some time).

To be honest though I don't actually know what this repository is being used for. Right now we have a copy of this in-tree (it's where they're being published from), but I don't think they're actively being maintained. Ideally we'd only have one copy of these guidelines? @steveklabnik do you know what the situation is here?